src/Resources/js/types/updatedParticipationCount.ts

```diff
@@ -0,0 +1,4 @@
+type UpdateParticipationCount = {
+    id: number,
```

Comment:
What id is this?

Reply:
Identificator for the meal which got updated, fixed naming